pkg/project/store/store.go

```diff
@@ -57,7 +57,7 @@ func (ps *projectStore) LoadProject(path string) (*domain.Project, error) {
 
 func (ps *projectStore) LoadProjectList(paths ...string) ([]*domain.Project, error) {
 	numPaths := len(paths)
-	if numPaths < 1 {
+	if numPaths < 1 && paths != nil {
```

Comment:
Passing `nil` to go's len function results in a length of 0, as you can see [here] (https://tour.golang.org/moretypes/12). Passing nil to the function `LoadProjectList` would thus make the if condition result in `true`, which would make the function return the return value of the function `loadAllProjects`.
So panics because of nil references should not occur here.